components/server/ee/src/billing/billing-mode.ts

```diff
@@ -135,12 +135,12 @@ export class BillingModesImpl implements BillingModes {
         // 3. Check team memberships/plans
         // UBB overrides wins if there is _any_. But if there is none, use the existing Chargebee subscription.
         const teamsModes = await Promise.all(teams.map((t) => this.getBillingModeForTeam(t, now)));
-        const hasUbbTeam = teamsModes.some((tm) => tm.mode === "usage-based");
+        const hasUbbPaidTeamSeat = teamsModes.some((tm) => tm.mode === "usage-based" && !!tm.paid);
         const hasCbTeam = teamsModes.some((tm) => tm.mode === "chargebee");
         const hasCbTeamSeat = cbTeamSubscriptions.length > 0;
 
-        if (hasUbbTeam || hasUbbPersonal) {
-            // UBB is gready: once a user has at least a team seat, they should benefit from it!
+        if (hasUbbPaidTeamSeat || hasUbbPersonal) {
+            // UBB is greedy: once a user has at least a team seat, they should benefit from it!
```

Comment:
Optional suggestion: Add "paid" team seat in the comment.

Reply:
Done.